core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -1610,12 +1610,12 @@ public PropServerConfiguration(
             if (pgEnabled) {
                 this.pgSelectCacheEnabled = getBoolean(properties, env, PropertyKey.PG_SELECT_CACHE_ENABLED, true);
                 final int effectivePGWorkerCount = pgWorkerCount > 0 ? pgWorkerCount : sharedWorkerCount;
-                this.pgSelectCacheBlockCount = getInt(properties, env, PropertyKey.PG_SELECT_CACHE_BLOCK_COUNT, 8 * effectivePGWorkerCount);
+                this.pgSelectCacheBlockCount = getInt(properties, env, PropertyKey.PG_SELECT_CACHE_BLOCK_COUNT, 32);
```

Comment:
With this change, we'll have 2048 cached factories with 32 workers - seems a bit high. Maybe we could also change the next line to something like
```java
this.pgSelectCacheRowCount = getInt(properties, env, PropertyKey.PG_SELECT_CACHE_ROW_COUNT, Math.max(effectivePGWorkerCount, 4));
```
?

This way the number will be 1024 instead of 2048. WDYT?

Reply:
fair